listenbrainz/spark/spark_reader.py

```diff
@@ -10,12 +10,20 @@
 from listenbrainz.db import user as db_user, stats as db_stats
 from listenbrainz.webserver import create_app
 from listenbrainz.db.exceptions import DatabaseException
+from listenbrainz.spark.handlers import handle_user_artist
 import sqlalchemy
 
 class SparkReader:
     def __init__(self):
         self.app = create_app() # creating a flask app for config values and logging to Sentry
 
+    def get_response_handler(self, response_type):
+        response_handler_map = {
```

Comment:
If wonder if we should declare the handler map at the top of the file for greater visibility?